### DISCUSSION TITLE
Modify commit message for config migration

### DISCUSSION THREAD

**Discussion by z0rc** [renovatebot/renovate#23877] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

Latest from https://gitlab.com/renovate-bot/renovate-runner

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Renovate's config.js has `configMigration: true`. I want to modify commit message for config migration commits to match our gitlab commit rules.

### What you tried so far.

I tried to look for suitable attribute in `packageRules` to match configuration migration events and edit `commitMessage` there, but wasn't able to find working solution.

Alternatively there is option `onboardingCommitMessage`, so maybe similar option can be introduced for config migration, like `configMigrationCommitMessage`.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
I suggest using a `RENOVATE_X_` variable within here: https://github.com/renovatebot/renovate/blob/a1f79bcf39f7dcf0951ed4260092ff8407dcfbbd/lib/workers/repository/config-migration/branch/commit-message.ts#L36C1-L38

**Reply by z0rc** [reply]
I'm reading https://docs.renovatebot.com/self-hosted-experimental/ and cannot find suitable variable. Do you suggest such variable should be added? Or am I missing something?

**Reply by rarkins** [reply]
I mean add a new variable for this purpose
